### PR TITLE
fix(config): Call debug log methods after setting the loglevel based upon config/cli-options

### PR DESCRIPTION
**Pull request by JakeChampion**
Currently the debug message about where the configfile was loaded from is always shown, regardless of the user defined loglevel. I've moved those debug messages to be done after the user's defined loglevel has been set which enables the user to choose whether they see this message or not :-)

Alternative fix the problem described in https://github.com/karma-runner/karma/pull/2658

### CODE REVIEW

File: lib/config.js [hunk 1]
```diff
@@ -389,12 +388,11 @@ var parseConfig = function (configFilePath, cliOptions) {
       return process.exit(1)
     }
   } else {
-    log.debug('No config file specified.')
     // if no config file path is passed, we define a dummy config module.
     configModule = function () {}
   }
 
-  var config = new Config()
+    var config = new Config()
```
Comment (dignifiedquire) [inline lib/config.js]: this seems unrelated/wrong indentation

Comment (JakeChampion) [reply lib/config.js]: Not quite sure how that happened, I've edited it via the Github editor.

**Comment by JakeChampion** [comment]
@dignifiedquire updated pr in line with your review

**Comment by dignifiedquire** [comment]
Thank you :octocat:
